_data/navigation.js

```diff
@@ -222,7 +222,7 @@ module.exports = {
                 url: '/docs/advanced-tutorial/',
               },
               {
-                title: 'Other Tutorials',
+                title: 'Other Resources',
```

Comment:
Let's move this one step out so it is not under Tutorials. This is a resources page now, so it doesn't fit specifically under Tutorials.